node-js-basics-101/node_modules/blob/test/index.js

```diff
@@ -0,0 +1,94 @@
+var Blob = require('../');
```

Comment:
__[jshint]__
* Redefinition of 'Blob'.

<!-- Comment by [SideCI](https://sideci.com) -->